modules/microshift-gitops-debug.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// microshift_running_apps/microshift-gitops.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-gitops-debug_{context}"]
+= Debugging GitOps on {microshift-short}
+
+There are two ways you debug GitOps applications in your {microshift-short} instance: Using sos reports 
+
+.Prerequisites 
+
+* You must have the `sos` package installed 
+
+.Procedure
+
+. Using sos reports to debug GitOps.
```

Comment:
We should probably refer to https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.15/html/support/microshift-sos-report here?